app/views/unattended/provisioning_templates/ZTP/vrp_default_ztp_config.erb

```diff
@@ -0,0 +1,57 @@
+<%#
+kind: ZTP
+name: VRP default ZTP config
+model: ProvisioningTemplate
+oses:
+- VRP
+-%>
+sysname <%= @host %> 
+#
+FTP server enable
+#
+dns resolve
+#
+stp mode rstp
+stp bpdu-protection
+#
+igmp-snooping enable
+#
+error-down auto-recovery cause bpdu-protection interval 120
+#
+aaa
+ authentication-scheme default
+ authorization-scheme default
+ accounting-scheme default
+ domain default
+ domain default_admin
+ undo local-user admin
+ local-user root password irreversible-cipher <%= Base64.decode64(root_pass) %>
+ local-user root privilege level 15
+ local-user root ftp-directory /
+ local-user root service-type terminal ssh ftp http
+#<%- @host.managed_interfaces.each do |interface| %>
+<%-   next if !interface.provision? || (interface.subnet.nil? && interface.subnet6.nil?) -%>
+vlan <%= interface.subnet.vlanid || 1 %>
+ description MGMT
+ management-vlan
+#
+interface Vlanif <%= interface.subnet.vlanid || 1 %>
+ ip address <%= interface.ip %> <%= interface.subnet.mask %>
+ description MGMT <%-if (interface.subnet.dns_primary != '') -%> 
+#
+dns server <%= interface.subnet.dns_primary %><%- end %>
+#<%- end %>
+user-interface con 0
+ authentication-mode aaa
+user-interface vty 0 4
+ authentication-mode aaa
+user-interface vty 16 20
+#
+nqa test-instance admin foreman
+ test-type <%= foreman_url('built')[/(.*):\/\//, 1] || 'http' %>
+ destination-address url <%= foreman_url('built')[/:\/\/([a-z0-9\-._~%]+|\[[a-z0-9\-._~%!$&'()*+,;=:]+\])/,1] %>
+ destination-port <%= foreman_url('built')[/:([0-9]+)\//,1] || 8000 %>
+ http-url <%= foreman_url('built') %> v1.1
+ start now
+#
+return
```

Comment:
Lemme take a look --- no I don't have any Huawei switches at home. I will trust you with this one, will just test rendering and deployment. Please file a PR into community-templates with this one.

If you want to make sure this doesn't break in the future, add this into `snapshots.yaml` and create a template snapshot via `bundle exec rake snapshots:generate RAILS_ENV=test`

Reply:
Okay I will move this template to community templates. I'll check out the second part of your comment, wasn't aware of this :-) thanks!

Reply:
Hmm, perhaps I've also misinterpreted that comment but, could it be that I **also** have to add this to community templates (so basically copy it there for others to be continue working on it?)

Reply:
The community templates are the main place the templates for Foreman live. We copy them to this repo before every release. So please add the template both here and at the community templates repo for now.

Reply:
Aaah, okay, clarifies a lot of things thank you @timogoebel, will do so